app/views/bets/_display_bets.html.erb

```diff
@@ -17,7 +17,15 @@
             <td class="col-md-2"><%= bet.created_at %></td>
             <td class="col-md-2"><%= bet.expires_at %></td>
             <td class="col-md-2"><%= bet.bet_total %></td>
-            <td class="col-md-4"><%= bet.bet %></td>
+            <td class="col-md-3"><%= bet.bet %></td>
+            <td class="col-md-1">
+              <%=
+                link_to bet, class: 'btn btn-secondary', role: 'button' do
+                  content_tag :i, class: 'fa fa-eye' do
+                  end
```

Comment:
Agreed.
To use content_tag without a block you have to pass it a third argument though: 
`content_tag :i, nil, class: 'stuff'`

In this case you shouldnt use content_tag. None of your values are dynamic. It would be more clear to write
`<i class="stuff"></i>`